src/renderers/shaders/ShaderChunk/common.glsl

```diff
@@ -4,6 +4,10 @@
 #define LOG2 1.442695
 #define EPSILON 1e-6
 
+#define square(a) a * a
```

Comment:
Is there an operator with greater precedence than "*"?   I think that there might be, but I am unsure.  May be best to put brackets around this as well.